frontend/src/App.js

```diff
@@ -1,24 +1,29 @@
-import logo from './logo.svg';
+// import logo from './logo.svg';
 import './App.css';
+import { SignUp } from './components/SignUp/signup';
+import { LogIn } from './components/LogIn/login';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    // <SignUp>
+    // </SignUp>
+    <LogIn></LogIn>
```

Comment:
It looks like you did this because you wanted to be able to see what the login page looks like. But eventually we'll going to need both the Login and Signup page to be accessible. The way we would solve something like that is by using something like React Router. 

Let's change this so that react router can set up routing for our pages.